src/Agent.Worker/TaskCommandExtension.cs

```diff
@@ -614,4 +675,11 @@ internal static class TaskSetTaskVariableEventProperties
         public static readonly String Variable = "variable";
         public static readonly String IsSecret = "issecret";
     }
+
+    internal static class TaskSetEndpointEventProperties
+    {
+        public static readonly String EndpointId = "endpointid";
```

Comment:
should we have the property as id (setendpoint command taking id seems more appropriate)?